roles/app/templates/env.j2

```diff
@@ -102,4 +102,18 @@ NEW_RELIC_APP_NAME="{{ new_relic_app_name }}"
 NEW_RELIC_LICENSE_KEY="{{ new_relic_license_key }}"
 {% endif %}
 
+{% if active_record_encryption_primary_key is defined %}
+ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY="{{ active_record_encryption_primary_key }}"
+{% endif %}
+{% if active_record_encryption_deterministic_key is defined %}
+ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY="{{ active_record_encryption_deterministic_key }}"
+{% endif %}
+{% if active_record_encryption_key_derivation_salt is defined %}
+ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT="{{ active_record_encryption_key_derivation_salt }}"
+{% endif %}
+
+{% if vine_api_url is defined %}
+VINE_API_URL="{{ vine_api_url }}"
+{% endif %}
```

Comment:
Do you think that a default would work instead?
```suggestion
VINE_API_URL="{{ vine_api_url | default('') }}"
```

Just a question, not a real suggestion.

Reply:
The code handle a missing `VINE_API_URL` with an error saying the VINE api isn't configured. With an empty string default it would result in a API connection error which could be confusing when the issue is actually a missing configuration.

Reply:
Ah yes, the code would need to check for the presence of the variable instead.